### PR TITLE
Uplift ubuntu to 22.04

### PR DESCRIPTION
**Pull request by mboukhalfa**
depends on https://github.com/metal3-io/metal3-dev-env/pull/1069

### CODE REVIEW

File: test/e2e/node_reuse_test.go [hunk 1]
```diff
@@ -42,7 +42,7 @@ func nodeReuse(clusterClient client.Client) {
 	)
 
 	const (
-		artifactoryURL = "https://artifactory.nordix.org/artifactory/metal3/images/k8s"
+		artifactoryURL = "dartifactory/metal3/images/k8s"
```
Comment (lentzi90) [inline test/e2e/node_reuse_test.go:45]: Typo?

Comment (mboukhalfa) [reply test/e2e/node_reuse_test.go]: thanks for caching that

**Review by lentzi90 — COMMENTED** [review]
/lgtm

**Comment by mboukhalfa** [comment]
/hold 
should merged together with https://github.com/metal3-io/metal3-dev-env/pull/1069

**Review by smoshiur1237 — APPROVED** [review]
/lgtm

**Review by furkatgofurov7 — APPROVED** [review]
Please run the tests when dev env is merged, thanks!

/approve

**Pull request merged**